<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hariki_and_tetrisClass</class>
 <widget class="QMainWindow" name="hariki_and_tetrisClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>874</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>hariki_and_tetris</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>630</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/shariki/tree-3358468.png);</string>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/shariki/test.png);</string>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="start_game">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать игру</string>
    </property>
   </widget>
   <widget class="QComboBox" name="time_game">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>5 минут</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10 минут</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15 минут</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20 минут</string>
     </property>
    </item>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>500</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/bg.png);
</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/bg.png);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/bg.png);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>190</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/bg.png);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/bg.png);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>590</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/bg.png);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_7">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>540</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/bg.png);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_8">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>480</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/bg.png);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_9">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>360</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/bg.png);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="hariki_and_tetris.qrc"/>
 </resources>
 <connections/>
</ui>
